src/pineko/scale_variations.py

```diff
@@ -0,0 +1,266 @@
+"""Module to generate scale variations."""
+import pathlib
+from typing import Tuple
+
+import numpy as np
+import pineappl
+import rich
+from eko import beta
+
+from . import check
+
+AS_NORM = 1.0 / (4.0 * np.pi)
+OrderTuple = Tuple[int, int, int, int]
+"""Tuple representing a PineAPPL order."""
+
+
+def sort_qcd_orders(order: OrderTuple) -> int:
+    """Define the sorting for an order. In particular the order is sorted according to the as order."""
+    return order[0]
+
+
+def ren_sv_coeffs(m, max_as, logpart, which_part, nf):
+    """Return the ren_sv contribution relative to the requested log power and perturbative order contribution (which_part).
+
+    Parameters
+    ----------
+    m : int
+        first non zero perturbative order
+    max_as : int
+        max order of alpha_s
+    logpart : int
+        power of the renormalization scale log asked
+    which_part : int
+        asked perturbative order contribution to be rescaled
+    nf : int
+        number of active flavors
+
+    Returns
+    -------
+    float
+        renormalization scale variation contribution
+    """
+    bcoeff = beta.beta_qcd((max_as - logpart - which_part + 2, 0), nf)
+    as_norm_fact = AS_NORM ** (max_as - which_part)
+    m_factor = m
```

Comment:
You can also use `m`: the modified variable would be only in this scope, since `float`s are not mutable (to first approximation, you can think that they don't own "dynamic memory").

The only effect of mutating `m` is that you won't have access any longer to the original value. But you don't need it any longer, since the rest of the function only uses `m_factor` and not `m`.
```suggestion
```

Reply:
[08aade5](https://github.com/NNPDF/pineko/pull/75/commits/08aade5443e9baeb6515e960d8234d0f2a418748)